agent/src/agent.php

```diff
@@ -76,6 +77,14 @@
 $debug = in_array($_SERVER['NIGHTWATCH_DEBUG'] ?? null, ['true', '1'], true);
 /** @var ?string $basePath */
 $basePath ??= str_replace(['phar://', '/agent.phar/src'], '', __DIR__);
+/** @var string|false $expectedSignature */
```

Comment:
For now, we can remove this as there isn't anything dynamic happening and the `$expectedSignature` can not be passed into the file, unlike the other variables.